handbook/engineering/developer-tools/index.md

```diff
@@ -0,0 +1,15 @@
+# Developer tools team
+
+**The developer tools team does not yet exist** and currently works as part of the [distribution team](../distribution/index.md).
+
+## Vision
+
+Create tools, systems and workflows that empower Sourcegraph engineers to work more efficiently and build quality products.
```

Comment:
This seems to have overlap with some of the goals the future [Backend Infrastructure team was supposed to have](https://about.sourcegraph.com/handbook/engineering/backend-infrastructure#vision). Is there a way to articulate this to remove this ambiguity? Or is there some actual overlap that we need to remove?

Reply:
Do you mean the vision or the goals below? I think both teams will likely have similar goals, but target different parts.
As I understand the backend team will focus more on internal libraries, while this team will focus on CI/CD and dev tooling.

Reply:
I meant the vision.

> I think both teams will likely have similar goals, but target different parts.

Agreed. Can we make it clear what those differences are in the Vision statement?

> As I understand the backend team will focus more on internal libraries

Libraries and shared infrastructure.

Reply:
> Libraries and shared infrastructure.

What type of shared infrastructure? Do you have something in mind?

> Can we make it clear what those differences are in the Vision statement?

Ill give it a look as I continue to update this document, is there a particular concern or a suggestion you have in mind? I dont think there is that much overlap to cause a problem at the moment.

Reply:
> What type of shared infrastructure? Do you have something in mind?

Examples: git-server and postgres.

> is there a particular concern or a suggestion you have in mind? I dont think there is that much overlap to cause a problem at the moment.

Just want to make sure we're aligned on these teams vision & long term goals and that its clear what their domains of ownership are. Would a candidate be able to understand the difference by reading the two teams pages?

Reply:
I believe they would.